sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -1637,25 +1613,27 @@ def message_content():
                     for message in messages:
                         print_message(_logger, message)
                         assert b''.join(message.body) == b'Test message'
-                        assert message.properties[b'key'] == b'value'
+                        assert message.application_properties[b'key'] == b'value'
                         assert message.content_type == 'application/text'
                         assert message.correlation_id == 'cid'
                         assert message.message_id == 'mid'
                         assert message.partition_key == 'pk'
-                        assert message.via_partition_key == 'via_pk'
                         assert message.to == 'to'
                         assert message.reply_to == 'reply_to'
                         assert message.time_to_live == timedelta(seconds=60)
 
-                        if message.label == '1st':
+                        if message.subject == '1st':
                             message_1st_received_cnt += 1
                             message.complete()
-                            message.label = '2nd'
+                            message.subject = '2nd'
                             sender.send_messages(message)  # resending received message
-                        elif message.label == '2nd':
+                        elif message.subject == '2nd':
                             message_2nd_received_cnt += 1
                             message.complete()
 
+                print("COUNTS&&&&&&&&&&&&&&&&&&&&")
+                print(message_1st_received_cnt)
+                print(message_2nd_received_cnt)
```

Comment:
emmm, should we remove these prints?

Reply:
HAH.  I need to create some sort of post-commit hook to tag and remove debug prints automatically for myself, I swear you catch them more than I do.  Removing.